### PR TITLE
Verifies arch in Dockerfile and polishes build

### PR DESCRIPTION
**Pull request by adriancole**
As we get more requests for architecture, we should be more careful and
at least lightly check what we intended to install.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -28,7 +28,13 @@ COPY --from=scratch /code/ .
 
 # Alpine's minirootfs is mirrored and only 5MB. Build on demand instead of consuming docker.io pulls
 WORKDIR /install
-# Use current version here: https://alpinelinux.org/downloads/
+
+# alpine_version is hard-coded here to allow the following to work:
+#  * `docker build https://github.com/openzipkin/docker-alpine.git`
```
Comment (adriancole) [inline Dockerfile:33]: note this rationale was missing before cc @jcchavezs @reta

File: Dockerfile [hunk 2]
```diff
@@ -38,6 +44,8 @@ FROM scratch as alpine
 ARG maintainer="OpenZipkin https://gitter.im/openzipkin/zipkin"
 LABEL maintainer=$maintainer
 LABEL org.opencontainers.image.authors=$maintainer
+ARG alpine_version=3.12.1
+LABEL alpine-version=$alpine_version
```
Comment (adriancole) [inline Dockerfile:48]: forgot to label the version before..

**Comment by adriancole** [comment]
Verify worked

```
 Verifying Docker image...
	/lib/ld-musl-x86_64.so.1 (0x7fa5c2920000)
	libc.musl-x86_64.so.1 => /lib/ld-musl-x86_64.so.1 (0x7fa5c2920000)
```

**Comment by adriancole** [comment]
remote build worked

```bash
$ docker build https://github.com/openzipkin/docker-alpine.git#verify-arch
[+] Building 9.2s (11/11) FINISHED                                                                                                                                                        
 => [internal] load git source https://github.com/openzipkin/docker-alpine.git#verify-arch                                                                                           2.1s
 => [internal] load metadata for ghcr.io/openzipkin/alpine:3.12.1                                                                                                                    0.0s
 => [install 1/5] FROM ghcr.io/openzipkin/alpine:3.12.1                                                                                                                              0.0s
 => [scratch 1/1] COPY . /code/                                                                                                                                                      0.0s
 => CACHED [install 2/5] WORKDIR /code                                                                                                                                               0.0s
 => [install 3/5] COPY --from=scratch /code/ .                                                                                                                                       0.0s
 => [install 4/5] WORKDIR /install                                                                                                                                                   0.0s
 => [install 5/5] RUN /code/alpine_minirootfs 3.12.1                                                                                                                                 1.5s
 => [alpine 1/2] COPY --from=install /install /                                                                                                                                      0.1s 
 => [alpine 2/2] RUN echo 'hosts: files mdns4_minimal [NOTFOUND=return] dns mdns4' >> /etc/nsswitch.conf && for repository in main testing community; do   repository_url=https://d  5.3s
 => exporting to image                                                                                                                                                               0.1s
 => => exporting layers                                                                                                                                                              0.1s
 => => writing image sha256:6449e2a7c871c66bd308e9e59be0bbe787eb0c4f8e068a7b9f97ece926715509
```

File: README.md [hunk 3]
```diff
@@ -1,30 +1,34 @@
...
+GitHub Container Registry: [ghcr.io/openzipkin/alpine](https://github.com/orgs/openzipkin/packages/container/package/alpine) includes:
  * `master` tag: latest commit
- * `N.M.L` tag: release
+ * `N.M.L` tag: release corresponding to a [Current Alpine Version](https://alpinelinux.org/downloads/)
 
-## Release process
-The Docker build is driven by `build-bin/build`. The argument to this must be Alpine's most specific
-version, ex `3.12.1` not `3.12`
- * You can look here: https://alpinelinux.org/downloads/
+## Using this the image
```
Comment (reta) [inline README.md:11]:
```suggestion
## Using this image
```

**Review by reta — APPROVED** [review]

**Pull request merged**